docs/proposals/device-crd.md

```diff
@@ -280,25 +319,20 @@ type DeviceStatus struct {
 // the actual state to the cloud. Offline device interaction in the edge is possible via twin
 // properties for control/command operations.
 type Twin struct {
-  // The property name for which the expected/actual values are reported.
+  // Required: The property name for which the expected/actual values are reported.
   // This property should be present in the device model.
-  // Required.
   Name     string    `json:"name,omitempty"`
-  // the expected attribute value for this property.
-  // Required.
+  // Required: the expected attribute value for this property.
   Expected Attribute `json:"expected,omitempty"`
-  // the actual attribute value for this property.
-  // Required.
+  // Required: the actual attribute value for this property.
   Actual   Attribute `json:"actual,omitempty"`
 }
 
 // Attribute represents the device property for which an Expected/Actual state can be defined.
 type Attribute struct {
-  // Describes whether this property is optional or not.
-  // Required.
+  // Required: Describes whether this property is optional or not.
   Optional string            `json:"optional,omitempty"`
```

Comment:
I was thinking if we really need this field or not. Does it make sense ?

Reply:
We don't need this field now